package.json

```diff
@@ -39,7 +41,7 @@
     "typescript": "^4.9.3"
   },
   "lint-staged": {
-    "*.{js,jsx,ts,tsx}": "eslint --cache --fix --max-warnings=0",
+    "*.{js,jsx,ts,tsx}": "eslint --cache --fix",
```

Comment:
nit: is it worth it to add explicit ignores for the lint rules we don't care about versus changing the global default to allow warnings?